modules/openideal_slideshow/templates/slideshow.html.twig

```diff
@@ -1,5 +1,77 @@
+{# The swiper slideshow. #}
 <div class="openideal-slideshow">
-    {% for item in items %}
-        <img src="{{ item.uri.value | image_style('slideshow_large') }}" alt="{{ item._referringItem.alt }}">
-    {% endfor %}
+    <ul class="swiper-wrapper my-gallery" itemscope itemtype="http://schema.org/ImageGallery">
+        {% for key, item in items %}
+            <li id="{{ key + 1 }}" class="swiper-slide" itemprop="associatedMedia" itemscope
+                itemtype="http://schema.org/ImageObject">
+                {# For the photoswipe use default image without styles. #}
+                <a id="{{ key == 0 ? 'first' : '' }}" title="click to zoom-in" href="{{ file_url(item.uri.value) }}"
+                   itemprop="contentUrl" data-size="1200x600">
+                    <img itemprop="thumbnail" src="{{ item.uri.value | image_style('slideshow_large') }}"
+                         alt="{{ item._referringItem.alt }}">
+                </a>
+            </li>
+        {% endfor %}
+    </ul>
+</div>
+
+
+{#
+  Add PhotoSwipe (.pswp) element to DOM -
+  Root element of PhotoSwipe. Must have class pswp.
+#}
+<div class="pswp" tabindex="-1" role="dialog" aria-hidden="true">
```

Comment:
1) Can we move this markup to a separate template and include it here?
2) t() function for labels

Reply:
1. Done